src/test/java/org/minions/devfund/abel/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,167 @@
+package org.minions.devfund.abel.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for Movie Database Unit Tests.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+
+    /**
+     * SetUp method.
+     */
+    @Before
+    public void setUp() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Test AddMovie.
+     */
+    @Test
+    public void testAddMovieNewMovieAndActors() {
+        final String expectedMovie = "NewMovie";
+        final String firstActor = "Actor1";
+        final String secondActor = "Actor2";
+        final int expectedmoviesSize = 1;
+        final int expectedActorsSize = 2;
+
+        movieDatabase.addMovie(expectedMovie, new String[]{firstActor, secondActor});
+        final int actualMoviesSize = movieDatabase.getMovieList().size();
+        final String actualMovie = movieDatabase.getMovieList().get(0).getName();
+        final int actualActorsSize = movieDatabase.getActorList().size();
+
+        assertEquals(expectedmoviesSize, actualMoviesSize);
+        assertEquals(expectedMovie, actualMovie);
+        assertEquals(expectedActorsSize, actualActorsSize);
+    }
+
+    /**
+     * Test AddMovie.
+     */
+    @Test
+    public void testAddMovieAlreadyAdded() {
+        final String expectedMovie = "NewMovie";
+        final String firstActor = "Actor1";
+        final String secondActor = "Actor2";
+        final int expectedMoviesSize = 1;
+        final int expectedActorsSize = 2;
+
+        movieDatabase.addMovie(expectedMovie, new String[]{firstActor, secondActor});
+        movieDatabase.addMovie(expectedMovie, new String[]{firstActor, secondActor});
+        final int actualMoviesSize = movieDatabase.getMovieList().size();
+        final String actualMovie = movieDatabase.getMovieList().get(0).getName();
+        final int actualActorsSize = movieDatabase.getActorList().size();
+
+        assertEquals(expectedMoviesSize, actualMoviesSize);
+        assertEquals(expectedMovie, actualMovie);
+        assertEquals(expectedActorsSize, actualActorsSize);
+    }
+
+    /**
+     * Test AddMovie.
+     */
+    @Test
+    public void testAddMovieNewActors() {
+        final String expectedMovie = "NewMovie";
+        final String secondMovie = "NewMovie2";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "NewMovie2" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)